lib/rspec/support/version_checker.rb

```diff
@@ -1,53 +0,0 @@
-module RSpec
-  module Support
-    LibraryVersionTooLowError = Class.new(StandardError)
-
-    # @private
-    class VersionChecker
```

Comment:
I think this is used outside of `rspec-support` (I'm thinking `rspec-rails` at least) so we might need to leave this in until we've switched those instances over.